src/scss/_svg.scss

```diff
@@ -18,14 +20,20 @@
   background-image: none;
   width: 12.5em; //200px
   position: absolute;
-  bottom: 4em; //if no javascript loads
+  bottom: 4em;
+  display: block;
+  left: 25%; //for browsers with no calc() support. Android 4.1.2
   left: calc(50% - 6.25em);
   z-index: 1;
 
   @media (min-width: $bp-e) {
     width: 18em;
     left: calc(50% - 9em);
```

Comment:
I see fallbacks for no `calc()` which is great. Should you provide one here as well?